startup_scripts/260_ip_addresses.py

```diff
@@ -3,63 +3,56 @@
 from virtualization.models import VirtualMachine
 from tenancy.models import Tenant
 from extras.models import CustomField, CustomFieldValue
-from ruamel.yaml import YAML
 
 from netaddr import IPNetwork
-from pathlib import Path
+from includes import load_yaml
 import sys
 
-file = Path('/opt/netbox/initializers/ip_addresses.yml')
-if not file.is_file():
-  sys.exit()
-
-with file.open('r') as stream:
-  yaml = YAML(typ='safe')
-  ip_addresses = yaml.load(stream)
+ip_addresses = load_yaml('/opt/netbox/initializers/ip_addresses.yml')
 
+if ip_addresses is not None:
```

Comment:
Maybe we can turn this `if` around:

```suggestion
if ip_addresses is None:
  sys.exit()
```

That would prevent a whole level of indention.